component/site/views/default/helpers/defaultloadedfromtemplate.php

```diff
@@ -1247,6 +1247,8 @@ function DefaultLoadedFromTemplate($view, $template_name, $event, $mask, $templa
 		$template_value = $tmprow->text;
 		$template_value = str_replace("@£@", "@", $template_value);
 
+		//We add new line characters again to avoid being marked as SPAM when using tempalte in emails
+		$template_value = preg_replace("@(<\s*(br)*\s*\/\s*(p|td|tr|table|div|ul|li|ol|dd|dl|dt)*?\s*>)+?@i","$1\n",$template_value);
```

Comment:
I'm not sure if I get your point. We just look for closing block tags and add a new line there. Having or not a > or < does not affect our purpose. Does it?